aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -61,20 +63,20 @@ export const InPageNavigation = ({ title }) => {
         height: 'calc(100vh - 50px)',
         overflowY: 'auto',
         position: 'sticky',
-        marginTop: '50px',
```

Comment:
I noticed we change this value, should we also change the "height" calculation a few lines above? is that `100vh - 50px` related to this 50px?

Reply:
Not related, the `height` calculation refers to the size of the box that contains the ToC, and I was playing around with it but I liked it where it was at before. We can discuss further on this.